cli/bpmetadata/types_ui_ext.go

```diff
@@ -0,0 +1,161 @@
+package bpmetadata
+
+type ExtensionType string
+
+const (
+	Undefined ExtensionType = "UNDEFINED_TYPE"
+
+	// General formats.
+	EmailAddress    ExtensionType = "EMAIL_ADDRESS"
+	MultiLineString ExtensionType = "MULTI_LINE_STRING"
+
+	// GCE related.
+	GceDiskImage       ExtensionType = "GCE_DISK_IMAGE"
```

Comment:
nit: acronyms capitalized here and throughout
```suggestion
	GCEDiskImage       ExtensionType = "GCE_DISK_IMAGE"
```

Reply:
done